bin/bootstrap

```diff
@@ -150,5 +139,8 @@ fi
 # Install oh-my-zsh
 sh -c "RUNZSH=no bin/oh-my-zsh.sh"
 
+# configure asdf for zsh
+# https://asdf-vm.com/guide/getting-started.html#_3-install-asdf
+echo -e "\n. $(brew --prefix asdf)/libexec/asdf.sh" >> ${ZDOTDIR:-~}/.zshrc
```

Comment:
Probably want to add a check to see if this line exists in the .zshrc file before appending it again.
I think most other commands in this script by default can be run more than once, but would be good to add a safety check here.